src/sentry/static/sentry/app/views/settings/project/projectCreateServiceHook.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {WithRouterProps} from 'react-router';
+
+import {t} from 'app/locale';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import ServiceHookSettingsForm from 'app/views/settings/project/serviceHookSettingsForm';
+
+type Props = WithRouterProps<{orgId: string; projectId: string}, {}>;
```

Comment:
I noticed that we usually use `RouteComponentProps`. should we use it here as well?